cleanhttp.go

```diff
@@ -18,23 +28,24 @@ func DefaultTransport() *http.Transport {
 		}).Dial,
 		TLSHandshakeTimeout: 10 * time.Second,
```

Comment:
I would explicitly set the `MaxIdleConnsPerHost` for clarity. Maybe just 1 is a sane default.